src/components/ResourceItem/tests/ResourceItem.test.tsx

```diff
@@ -215,6 +216,48 @@ describe('<ResourceItem />', () => {
     });
   });
 
+  describe('external', () => {
+    it('renders an <UnstyledLink /> with no external prop ', () => {
+      const element = mountWithAppProvider(
+        <ResourceListContext.Provider value={mockDefaultContext}>
+          <ResourceItem id="itemId" url={url} />
+        </ResourceListContext.Provider>,
+      );
+
+      expect(element.find(UnstyledLink).exists()).toBe(true);
```

Comment:
Should we try and have an assertion that shows the `external` prop isn't set to `true` by default to match the test description?